blueprints/ember-z-schema/index.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  afterInstall: function () {
+    return this.addPackagesToProject([
+      {name: 'z-schema'}
+    ])
+  },
+
+  normalizeEntityName: function () {
+    // this prevents an error when the entityName is
+    // not specified (since that doesn't actually matter
+    // to us
```

Comment:
Closing paren? Also, this could probably all go on one line, did you copy it from somewhere?